.gitignore

```diff
@@ -24,3 +24,4 @@ dist-ssr
 *.sw?
 
 /*/.env
+images
```

Comment:
@Saurabhchaudhary9799 why did you add images folder in .gitignore? IF it is for local setup please remove it

Reply:
@Saurabhchaudhary9799  ^^